basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/ParametersFactory.java

```diff
@@ -0,0 +1,68 @@
+package org.jboss.hal.testsuite.test.configuration.infinispan;
+
+import org.apache.commons.lang.RandomStringUtils;
+import org.jboss.hal.testsuite.test.configuration.infinispan.cache.container.CacheContainerContext;
+import org.jboss.hal.testsuite.test.configuration.infinispan.cache.container.CustomCacheContainerContext;
+import org.jboss.hal.testsuite.test.configuration.infinispan.cache.container.DefaultCacheContainer;
+import org.jboss.hal.testsuite.test.configuration.infinispan.cache.type.DistributedCacheContext;
+import org.jboss.hal.testsuite.test.configuration.infinispan.cache.type.InvalidationCacheContext;
+import org.jboss.hal.testsuite.test.configuration.infinispan.cache.type.LocalCacheContext;
+import org.jboss.hal.testsuite.test.configuration.infinispan.cache.type.ReplicatedCacheContext;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+public class ParametersFactory {
+
+    private OnlineManagementClient client;
+
+    public ParametersFactory(OnlineManagementClient client) {
+        this.client = client;
+    }
+
+    public Collection cacheContainerTable() {
```

Comment:
Please add JavaDoc describing the purpose of this method as well as the structure of the provided collection.